protocol/x/accountplus/keeper/keeper.go

```diff
@@ -62,6 +63,40 @@ func (k Keeper) SetGenesisState(ctx sdk.Context, data types.GenesisState) error
 	return nil
 }
 
+func (k Keeper) InitializeAccount(ctx sdk.Context, address sdk.AccAddress) (types.AccountState, error) {
+	store := ctx.KVStore(k.storeKey)
+
+	if _, found := k.getAccountState(store, address); found {
+		return types.AccountState{}, errors.New(
+			"Cannot initialize AccountState for address with existing AccountState, address: " + address.String(),
+		)
+	}
+
+	initialAccountState := types.AccountState{
+		Address:               address.String(),
+		TimestampNonceDetails: DeepCopyTimestampNonceDetails(InitialTimestampNonceDetails),
```

Comment:
Let's make the [field definition](https://github.com/dydxprotocol/v4-chain/blob/e86691c003285eb0d00babeadfb8d7828fedc150/proto/dydxprotocol/accountplus/accountplus.proto#L9) not nullable ([eg](https://github.com/dydxprotocol/v4-chain/blob/e86691c003285eb0d00babeadfb8d7828fedc150/proto/dydxprotocol/clob/tx.proto#L78)) so this is never a pointer.

Reply:
If the field is not nullable, will go automatically copy by value?

Reply:
I believe structs are copied by value but if the struct has a slice field, the copy's slice will reference the same slice.

Reply:
The nullable comment is not just for deep copy, but more for proto conventions (since dealing with pointers is annoying)

Reply:
I'm also wondering why we need to keep this `InitialTimestampNonceDetails` at all. As asked above, when would we ever want to write an empty account state to [state](https://github.com/dydxprotocol/v4-chain/pull/1941/files#r1687176199)?